rabbit-escape-ui-android/app/src/main/java/rabbitescape/ui/android/AndroidCanvas.java

```diff
@@ -40,6 +43,24 @@ public void drawLine(
         float startX, float startY, float stopX, float stopY, AndroidPaint paint
     )
     {
-        this.canvas.drawLine( startX, startY, stopX, stopY, paint.paint );
+        this.canvas.drawLine(startX, startY, stopX, stopY, paint.paint);
+    }
+
+    @Override
+    public void drawFilledPoly( int[] xs, int[] ys, AndroidPaint paint )
```

Comment:
the android Path class is huge and powerful. It does not seem reasonable to attempt to duplicate it in swing when we dont need it. calling it addPath would give the wrong impression. All we want to do is a small bit of what Path can do: polygons.